test/aqua/test_optimizers.py

```diff
@@ -22,15 +22,16 @@
 
 from qiskit.aqua import aqua_globals
 from qiskit.aqua.components.optimizers import (ADAM, CG, COBYLA, L_BFGS_B, P_BFGS, NELDER_MEAD,
-                                               POWELL, SLSQP, SPSA, TNC)
+                                               POWELL, SLSQP, SPSA, TNC, GSLS)
 
 
 class TestOptimizers(QiskitAquaTestCase):
     """ Test Optimizers """
+
     def setUp(self):
         super().setUp()
         aqua_globals.random_seed = 50
-        pass
+        np.random.seed(512310912)
```

Comment:
We should not have to do this for aqua modules. See my comment in the GSLS optimizer above to change its use over to aqua globals random rather than going direct to numpy.